interpreter.go

```diff
@@ -723,14 +723,14 @@ func serializeJSON(v interface{}, multiline bool, indent string, buf *bytes.Buff
 	}
 }
 
-func (i *interpreter) manifestAndSerializeJSON(trace *TraceElement, v value, multiline bool, indent string) (string, error) {
+func (i *interpreter) manifestAndSerializeJSON(trace *TraceElement, v value, multiline bool, indent string) (*bytes.Buffer, error) {
```

Comment:
Maybe it would be better to take this buffer as an argument. Returning a buffer seems weird.

Reply:
That, or even more generally, demand an `io.Writer`. The downside then is all the potential errors you have to deal with, whereas with a `bytes.Buffer` the operations cannot fail.